src/main/java/emissary/core/BaseDataObject.java

```diff
@@ -180,7 +180,7 @@ public class BaseDataObject implements Serializable, Cloneable, Remote, IBaseDat
     /**
      * A factory to create channels for the referenced data.
      */
-    protected SeekableByteChannelFactory seekableByteChannelFactory;
+    protected SeekableByteChannelFactory<?> seekableByteChannelFactory;
```

Comment:
SCBF references in the BaseDataObject class and the IBDO interface use the unbounded wildcard type specifier (`<?>`), since the BDO itself should have no need to require a specific `SeekableByteChannel` implementation, exactly as it had been before the `SeekableByteChannelFactory` interface was made generic.